apps/client/src/common/components/view-params-editor/ViewParamsEditor.tsx

```diff
@@ -96,10 +99,7 @@ export default function ViewParamsEditor({ paramFields }: EditFormDrawerProps) {
 
         <DrawerFooter className={style.drawerFooter}>
           <Button variant='ontime-ghosted' onClick={resetParams} type='reset'>
-            Reset
-          </Button>
-          <Button variant='ontime-subtle' onClick={onCloseWithoutSaving}>
-            Cancel
+            Reset to default
           </Button>
           <Button variant='ontime-filled' form='edit-params-form' type='submit'>
             Save
```

Comment:
did you want to call this submit?

Reply:
I didn't understand the question.. is there something with the naming?

Reply:
Ah yes, sorry.

I tried it and it felt a bit weird. 
The idea of submit was to clarify the expectation of the window disappearing on click, but I think it made the language more complicated.
What do you think?

Reply:
both are OK with me
if you think its a bit wired let's not do it